src/scales/scale.time.js

```diff
@@ -360,8 +360,9 @@ module.exports = function(Chart) {
 		getLabelForIndex: function(index, datasetIndex) {
 			var me = this;
 			var label = me.chart.data.labels && index < me.chart.data.labels.length ? me.chart.data.labels[index] : '';
+			var firstData = me.chart.data.datasets[datasetIndex].data[0];
```

Comment:
@simonbrunel, you're right. That should be index there, i'll go ahead and change that.